### ISSUE TITLE
Introductory tutorial: `@threlte/extras`

### ISSUE THREAD

**Issue by ixxie** [threlte/threlte#547]
Each package should have a solid introductory tutorial, analogous to `@threlte/core`'s [Your First Scene](https://threlte.xyz/docs/learn/getting-started/your-first-scene).

`@threlte/extras` has too many elements to cover all of them, so this tutorial should select some highlights. The tutorial should have a coherent narrative: the learner should be making a scene that makes sense, and learning about `@threlte/extras` as they build it.

If you consider contributing this, please discuss your plan here before submitting a PR.

**Comment by DefinitelyMaybe** [comment]
We did discuss this some time ago and iirc we thought extras should stay a melting pot of elements. When folks get down into the weeds enough - and want more dedicated packages - thats when things might be peeled off.

Just wanted to add this comment to colour in some context for any further discussion.

We could close the issue if it no longer suits but iirc there wasn't 100% agreement on this.

**Comment by michealparks** [comment]
@DefinitelyMaybe I agree. Closing this unless someone wants to advocate for it. I like the idea of intros for most Threlte repos, but extras doesn't make much sense to me.
